client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -194,6 +201,119 @@ export default function CompositeCheckout( {
 		);
 	};
 
+    function getTermText( term ) {
+        switch ( term ) {
+            case TERM_BIENNIALLY:
+                return translate( 'Two years', {
+                    context: 'subscription length',
+                } );
+
+            case TERM_ANNUALLY:
+                return translate( 'One year', {
+                    context: 'subscription length',
+                } );
+
+            case TERM_MONTHLY:
+                return translate( 'One month', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.